moped-editor/src/views/projects/projectView/ProjectNotes.js

```diff
@@ -90,6 +90,9 @@ const useStyles = makeStyles((theme) => ({
 // The zeroth item in the list is intentionally blank; the notes are 1-indexed.
 const projectNoteTypes = ["", "Internal Note", "Status Update"];
 
+export const STATUS_UPDATE_TYPE_ID = 2;
+export const INTERNAL_NOTE_TYPE_ID = 1;
```

Comment:
thank you for centralizing these values. i will open an issue to get them into a lookup table in the DB